cmake/ecbuild_add_test.cmake

```diff
@@ -459,7 +459,7 @@ macro( ecbuild_add_test )
 
       if( EC_OS_NAME MATCHES "windows" AND ${_PAR_TYPE} MATCHES "SCRIPT" )
         # Windows has to be explicitly told to use bash for the tests.
-        set( _WIN_CMD "bash" "--rcfile" "${CMAKE_CURRENT_SOURCE_DIR}/windows_testing.bashrc" "-ci" )
+        set( _WIN_CMD "bash" "--rcfile" "${WINDOWS_TESTING_BASHRC_DIR}/windows_testing.bashrc" "-ci" )
```

Comment:
Instead of letting users change the directory, the recommended practise is for users to change the whole path to the file. This provided more control, including in the name of the file. Solution is therefore to provide a default value, but let users override it.

Reply:
Done